openshift/cronJob-template.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  annotations:
+    thoth-station.ninja/template-version: 0.7.0
+  labels:
+    app: srcopsmetrics
+  name: srcopsmetrics
+spec:
+  schedule: "0 0 * * 0"
+  jobTemplate:
+    metadata:
+      name: srcopsmetrics
+    spec:
+      template:
+        spec:
+          containers:
+          - name: srcopsmetrics
+            image: docker-registry.default.svc:5000/dtuchyna-thoth-dev/srcopsmetrics
+            env:
+              # - name: APP_FILE
+              #   value: ./srcopsmetrics/cli.py
+              - name: GITHUB_ACCESS_TOKEN
+                valueFrom:
+                  secretKeyRef:
+                    name: srcopsmetrics-secrets
+                    key: githubAccessToken
+              - name: THOTH_CEPH_SECRET_KEY
+                valueFrom:
+                  secretKeyRef:
+                    name: srcopsmetrics-secrets
+                    key: thothCephSecretKey
+              - name: THOTH_CEPH_KEY_ID
+                valueFrom:
+                  secretKeyRef:
+                    name: srcopsmetrics-secrets
+                    key: thothCephKeyID
+              - name: PYTHONPATH
+                value: "."
+              - name: PREFIX
+                value: data/thoth/dtuchyna
+              - name: HOST
```

Comment:
These values can be taken from thoth ConfigMap :)

Reply:
I created my own srcopsmetrics ConfigMap, is that right?

Reply:
I would reuse as much as possible what we have already if possible :)